README.rst

```diff
@@ -14,103 +14,69 @@ Microsoft Azure SDK for Python
 
-- Specify the version:  ``pip install azure-mgmt-compute==0.30.0rc6``
 
-If you want to install ``azure`` from source::
+You can also install a set of Azure libraries in a single line using the ``azure`` meta-package.
+Since not all packages in this meta-package are
+published as stable yet, the ``azure`` meta-package is still in release candidate.
+However, the core packages, from code quality/completeness perspectives can at this time be considered "stable" 
+- it will be officially labeled as such in sync with other languages as soon as possible. 
+We are not planning on any further major changes until then.
 
-    git clone git://github.com/Azure/azure-sdk-for-python.git
-    cd azure-sdk-for-python
-    python setup.py install
+**Since it's a preview release, you need to use the ``--pre`` flag:**
 
-DISCLAIMER
-==========
+.. code-block:: console
 
-This is a release candidate. However, the core packages, from code quality/completeness perspectives can at this time be considered "stable" - 
-it will be officially labeled as such in September (in sync with other languages).
+   $ pip install --pre azure
+   
+or directly
 
-We are not planning on any further major changes until then.
+.. code-block:: console
 
-The following packages are still labeled "preview" but can be considered "stable":
-
-- azure-mgmt-resource 0.31.0
-- azure-mgmt-compute 0.33.0
-- azure-mgmt-network 0.30.0
-- azure-mgmt-storage 0.31.0
-- azure-mgmt-keyvault 0.30.1
-
-The following packages are already released as "stable" and are officially production ready:
-
-- azure-batch 2.0.0
-- azure-mgmt-batch 3.0.0
-- azure-mgmt-devtestlabs 1.0.0
-- azure-mgmt-dns 1.0.0
-- azure-mgmt-logic 1.0.0
-- azure-mgmt-redis 4.0.0
-- azure-mgmt-scheduler 1.1.1
-- azure-mgmt-servermanager 1.1.0
-- azure-servicebus 0.21.0
-- azure-servicemanagement-legacy 0.20.5
-- azure-storage 0.33.0
-
-The following packages are also available as preview only, not ready for production,
-and will NOT be installed with the 2.0.0rc6 "azure" meta-package. We removed then from the 2.0.0rc6
-to prepare our customers to the 2.0.0 stable release that will only contains the stable packages
-listed before.
-
-- azure-keyvault 0.1.0
-- azure-graphrbac 0.30.0rc6
-- azure-monitor 0.2.0
-- azure-mgmt-authorization 0.30.0rc6
-- azure-mgmt-cdn 0.30.2
-- azure-mgmt-cognitiveservices 0.30.0rc6
-- azure-mgmt-containerregistry 0.1.1
-- azure-mgmt-datalake-analytics 0.1.3
-- azure-mgmt-datalake-store 0.1.3
-- azure-mgmt-documentdb 0.1.0
-- azure-mgmt-commerce 0.30.0rc6
-- azure-mgmt-eventhub 0.2.0
-- azure-mgmt-iothub 0.2.1
-- azure-mgmt-media 0.1.1
-- azure-mgmt-monitor 0.1.0
-- azure-mgmt-notificationhubs 0.30.0
-- azure-mgmt-powerbiembedded 0.30.0rc6
-- azure-mgmt-search 0.1.0
-- azure-mgmt-servicebus 0.1.0
-- azure-mgmt-sql 0.3.0
-- azure-mgmt-trafficmanager 0.30.0rc6
-- azure-mgmt-web 0.31.0
+   $ pip install azure==2.0.0rc6
 
+**The azure meta-package 1.0.3 is deprecated and is not working anymore.**
```

Comment:
"**The azure meta-package 1.0.3 is deprecated and no longer properly functions.**"